app/workers/podcast_episodes/create_worker.rb

```diff
@@ -0,0 +1,11 @@
+module PodcastEpisodes
+  class CreateWorker
+    include Sidekiq::Worker
+
+    sidekiq_options queue: :high_priority
```

Comment:
Should we explicitly add some retries? cc: @mstruve 

```ruby
sidekiq_options queue: :high_priority, retry: 10
```

Reply:
@atsmith813 by default Sidekiq will try every job 25 times over the course of 30 days. I think leaving this one is fine. Anything with external requests as I found out with an analytics job recently we want to allow max retries in the event of throttling. Anything that is just doing internal work is fine to keep to shorter. I'd rather have it retry too long than no enough honestly so leaving it blank is 👍 from me

Reply:
ah true, throttling is a good use case to set a max limit 👍